### ISSUE TITLE
Remove "Termination" from the properties to be model-checked

### ISSUE THREAD

**Issue by sophiaIC** [Theodus/backpressure-model#13]
I thought we were going to replace in PROPERTIES the requirement  `Termination` by `true`, and add a further invariant that says
      `\exists c. Run( c )   \/    Unmute  \/  Terminating`

The above formulation is equivalent, but I hope is cheaper to check as it does not involve any temporal operators. @Theodus, is it possible to compare the performance of the two versions

**Comment by lukecheeseman** [comment]
I agree, we are more interested in progress than termination. We already have the expensive temporal spec property being checked.

I don't believe we need Sophia's proposed:
```
\exists c. Run( c ) \/ Unmute \/ Terminating
```
This invariant is the `Next` relation, a temporal property already checked (and used to compute the next state?), in the model:
```
Terminating ==
  /\ \A c \in Cowns: Sleeping(c)
  /\ UNCHANGED vars
Next == Terminating \/ \E c \in Cowns: Run(c) \/ Unmute
Spec ==
  /\ Init
  /\ [][Next]_vars
  /\ \A c \in Cowns: WF_vars(Run(c))
  /\ WF_vars(Unmute)
```

Aside: I think this reads more like a progress lemma when written as:
```
Next ==  \E c \in Cowns: Run(c) \/ Unmute \* remove terminating from next
Spec ==
  /\ Init
  /\ [][Next \/ Terminating]_vars         \* add terminating here
  /\ \A c \in Cowns: WF_vars(Run(c))
  /\ WF_vars(Unmute)
```

**Comment by Theodus** [comment]
Terminating has been moved from `Next` to `Spec` as @lukecheeseman suggested: https://github.com/Theodus/backpressure-model/commit/a65f77ccfb306ed2b623614a7272c9e0c8f17042
